server/src/server.ts

```diff
@@ -120,7 +121,7 @@ connection.onRequest(RequestMethod.getLinksInDocument, (params: RequestParams['g
 
 connection.listen()
 
-documents.onDidChangeContent(async (event) => {
+const analyzeDocument = async (event: TextDocumentChangeEvent<TextDocument>): Promise<void> => {

```

Comment:
not sure this name describes properly the function, but I am not convinced onDidOpenOrChangeContent would be better, so whatever.